ros2topic/ros2topic/verb/echo.py

```diff
@@ -151,11 +153,18 @@ def subscribe_and_spin(
                 event_callbacks=event_callbacks,
                 raw=raw)
         except UnsupportedEventTypeError:
-            assert report_lost_messages
```

Comment:
nitpick: `assert not no_report_lost_messages` can stay here? it expects `UnsupportedEventTypeError` only if `QoSEventHandler` cannot be created from rmw.

Reply:
Addressed on [b2b222c](https://github.com/gonzodepedro/ros2cli/commit/b2b222ccf57bd51fbbd0295269a25a398b1e94a7)

I also removed the print right after it to silently continue.

Reply:
> I also removed the print right after it to silently continue.

i am okay with this but i would prefer to have it to let user know that rmw cannot deliver the report of lost messages. i guess this is informative for user.

Reply:
IMO, I would silently continue. I think it's probably likely that many RMWs probably don't support this feature and it would be annoying to see this message every time you run `ros2 topic echo`.

Reply:
that actually makes sense too. i am okay with either way.